src/transactions/feed/JumpstartTransactionDetailsScreen.tsx

```diff
@@ -238,6 +312,22 @@ function JumpstartTransactionDetailsScreen({ route }: Props) {
           style={styles.tokenFiatValueContainer}
           textStyle={styles.tokenFiatValueText}
         />
+        {error && (
```

Comment:
i wondered if it's worth it to surface a different error message if the user doesn't have enough gas for reclaim....it's probably okay with a generic message for now